tiago/todoList/main.py

```diff
@@ -0,0 +1,37 @@
+import webapp2
+import json
+import datetime
+from google.appengine.api import users
+
+from models import *
+from utils import *
+
+class BaseHandler(webapp2.RequestHandler):
+    pass
+
+class LoginHandler(BaseHandler):
+    def get(self):
+        user = users.get_current_user()
+        if user:
```

Comment:
O handler de login deve fazer apenas o login. A parte de logout já está sendo feita no handler de logout, note que há repetição de código nos dois handlers. Sugestão, crie um decorator que verifique se o usuário está logado, assim não será repetir o código de verificação toda vez que for usá-lo.